core-primitives/types/Cargo.toml

```diff
@@ -16,7 +16,6 @@ sgx_tstd = { rev = "v1.1.3", features = ["untrusted_fs","net","backtrace"], git
 
 # local deps
 itp-storage = { path = "../storage", default-features = false }
-its-primitives = { path = "../../sidechain/primitives", default-features = false }
```

Comment:
I think this is a good change, core-primitives should be the bottom of the dependency hierarchy (at least in my understanding), so they shouldn't depend on any higher level crates (and side-chain imo is higher level).

Reply:
Yes, I totally agree.